lib/catalog/approval_transition.rb

```diff
@@ -43,7 +43,7 @@ def cancel_all
       level = @state == "Failed" ? "error" : "info"
       @order_item.order.order_items.each do |item|
         state = item == @order_item ? @state : "Canceled"
-        @order_item.update_message(level, "Order Item #{state}")
+        item.update_message(level, "Order Item #{state}")
         Rails.logger.send(level, "Order Item #{item.id} marked as #{state} because approval status is #{@state}")
```

Comment:
Do you think we should change the `update_message` to also include more information about why it was marked as that state, similar tot he log message? I'm just thinking about seeing "Order Item Canceled" versus "Order Item 123 marked as Canceled because approval status is Denied" seems like it might make more sense.